index.html

```diff
@@ -16,6 +16,7 @@ <h2>Our Advice for Newbies!</h2>
     </header>
 
     <section>
+      <a href="Tammala-Kennedy-advice.html"> Tammala's word of advice</a><br>
```

Comment:
Hi @TammKenn,

Great start here! Can you change this link so that it goes to the advice folder first, then to your advice file? And can you change the name of the advice file in the link so that it's all lowercase letters to match the file name? Thanks!